ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -41,13 +45,31 @@ Install RHODS
            IF  "${UPDATE_CHANNEL}" != "odh-nightlies"
                 Install RHODS In Managed Cluster Using CLI  ${cluster_type}     ${image_url}
            ELSE
+                ${operator_csv_name} =    Set Variable    ${ODH_OPERATOR_NAME}
                 Create Catalog Source For Operator
                 Oc Apply    kind=List    src=tasks/Resources/Files/odh_nightly_sub.yml
            END
       ELSE
           FAIL    Provided test envrioment is not supported
       END
   END
+  Wait Until Csv Is Ready    ${operator_csv_name}
+
+Wait Until Csv Is Ready
+  [Documentation]   Waits some time for given CSV to be in Succeeded status condition
+  [Arguments]    ${csv_name}
+  Log    Waiting for the '${csv_name}' operator CSV in 'Succeeded' status condition    console=yes
+  Wait Until Keyword Succeeds    6 times   20 seconds
+  ...    Csv Is Ready    ${csv_name}
+  Log    Operator '${csv_name}' CSV is in 'Succeeded' status condition now, let's continue    console=yes
+
+Csv Is Ready
+  [Documentation]   Check whether given CSV to be in Succeeded status condition
+  [Arguments]    ${csv_name}
+  ${rc}    ${output} =    Run And Return Rc And Output
+  ...    oc get csv --namespace openshift-operators --output=json | jq --raw-output --exit-status '.items[] | select(.metadata.name | test("${csv_name}")).status.conditions[] | select(.phase == "Succeeded").phase'    # robocop: disable:line-too-long
+  Should Be Equal As Integers    ${rc}    0
+  Should Be Equal As Strings    Succeeded    ${output}
```

Comment:
we have kw which check the subscription and looks for `AllCatalogSourcesHealthy` condition. Isn't it equivalent?

```
    Wait Until Operator Subscription Last Condition Is
    ...    type=CatalogSourcesUnhealthy    status=False
    ...    reason=AllCatalogSourcesHealthy    subcription_name=${SUBSCRIPTION_NAME}
    ...    namespace=${NAMESPACE}
```

Reply:
Yeah, TBH, I'm not really sure what is the proper way here. I saw that the subscription marks `CatalogSourcesUnhealthy = false` earlier than the CSV sets `Succeeded = InstallSucceeded` with message `install strategy completed with no errors`. Thus, I though that better way how to check for this is actually checking CSV status. But maybe I'm wrong here, please point me to some source of truth :shrug:

Reply:
hmm I imagine it's a matter of microseconds/seconds? anyways, okay.

Will you have another PR to actually use the keyword?

Reply:
Well, the difference is definitely at least seconds. In case of ODH, it looks like the installation take a bit more time than for RHOAI for some reason, where we didn't hit this issue :shrug:

I don't plan to use the other keyword here or this one on other places unless we face an issues, I guess :slightly_smiling_face: